tools/cxbe/Xbe.cpp

```diff
@@ -992,7 +1009,12 @@ void Xbe::Export(const char *x_szXbeFilename)
 
             if(fwrite(m_bzSection[v], RawSize, 1, XbeFile) != 1)
             {
-                sprintf(szBuffer, "Unexpected write error while writing Xbe Section %d (%Xh) (%s)", v, v, m_szSectionName[v]);
+                snprintf(szBuffer,
+                         sizeof(szBuffer),
+                         "Unexpected write error while writing Xbe Section %d (%Xh) (%s)",
+                         v,
+                         v,
+                         m_szSectionName[v]);
```

Comment:
nit: All of these edits are for `SetError` which internally has a maximum length anyway.
I think this change is good, but we should probably fix the underlying issue by having `SetError(bool x_bFatal, const char *x_szErrorFmt, ...)` which simplifies the code further.